src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -351,7 +350,7 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
     public String getQuestionResultStatisticsHtml(
             List<FeedbackResponseAttributes> responses,
             FeedbackQuestionAttributes question,
-            PageData pageData,
+            String studentEmail,
```

Comment:
It looks odd that a studentEmail is passed into getQuestionResultStatisticsHtml, which is used on both instructor and student pages. Maybe add a comment, or make a wrapper method for instructor pages? If so, should be done in another issue.